### PR TITLE
Make Julia maintainers consistent with docs

### PR DESCRIPTION
**Pull request by simonbyrne**
cc: @ararslan @staticfloat @StefanKarpinski

### CODE REVIEW

File: lib/travis/build/script/julia.rb [hunk 1]
```diff
@@ -2,10 +2,9 @@
 
 # Community maintainers:
 #
-#   Tony Kelman       <tony kelman net, @tkelman>
-#   Pontus Stenetorp  <pontus stenetorp se, @ninjin>
-#   Elliot Saba       <staticfloat gmail com, @staticfloat>
-#   Simon Byrne       <simonbyrne gmail.com, @simonbyrne>
+#   Alex Arslan       (@ararslann)
```
Comment (ararslan) [inline lib/travis/build/script/julia.rb]:
```suggestion
#   Alex Arslan       (@ararslan)
```

**Comment by BanzaiMan** [comment]
We can merge this once everyone involved signs off on it. And, thank you for your contribution!

**Comment by BanzaiMan** [comment]
@tkelman @ninjin

**Comment by StefanKarpinski** [comment]
@tkelman is no longer an active Julia maintainer and is unlikely to respond. @ninjin will likely respond.

**Comment by tkelman** [comment]
I'm not likely to want to respond when someone mistakenly pings me on a base Julia issue or PR, but I don't mind being left on this list. It's low volume and I still know how this was implemented and is supposed to work.

**Comment by ninjin** [comment]
@BanzaiMan I am fine with being taken off the list, I would bet that @tkelman is more likely to respond more promptly than myself as my recent faculty promotion keeps me properly bogged down. '^^

**Comment by staticfloat** [comment]
In case there is any question, I’m okay with being on this list.

**Comment by BanzaiMan** [comment]
@tkelman The PR removes you at the moment. If you would like to be back on the list, I can put you back in. Let me (or @simonbyrne) know.

**Comment by StefanKarpinski** [comment]
This is fine, please go ahead and merge.

**Comment by tkelman** [comment]
No it isn't, I'd rather be left on.

**Comment by StefanKarpinski** [comment]
Please go ahead and make a PR adding your name back.

**Comment by StefanKarpinski** [comment]
As a policy question for @BanzaiMan: does the project decide who goes on this list or can any individual who wants to be added to this list make a PR adding themselves?

**Comment by BanzaiMan** [comment]
I consider the maintainers a team, and I would like a consensus built. :-)

**Pull request merged**